app/src/main/java/com/teamcatchme/calendar_customview/utils.kt

```diff
@@ -0,0 +1,23 @@
+package com.teamcatchme.calendar_customview
+
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+
+
+fun drawableToBitmap(drawable: Drawable): Bitmap? {
+    if (drawable is BitmapDrawable) {
+        return drawable.bitmap
+    }
+    val bitmap =
+        Bitmap.createBitmap(
+            drawable.intrinsicWidth,
+            drawable.intrinsicHeight,
+            Bitmap.Config.ARGB_8888
+        )
+    val canvas = Canvas(bitmap)
+    drawable.setBounds(0, 0, canvas.getWidth(), canvas.getHeight())
+    drawable.draw(canvas)
+    return bitmap
+}
```

Comment:
```suggestion
fun Drawable.ToBitmap(): Bitmap? {
    if (this is BitmapDrawable) {
        return bitmap
    }
    val bitmap =
        Bitmap.createBitmap(
            intrinsicWidth,
            intrinsicHeight,
            Bitmap.Config.ARGB_8888
        )
    val canvas = Canvas(bitmap)
    setBounds(0, 0, canvas.getWidth(), canvas.getHeight())
    draw(canvas)
    return bitmap
}
```
이런식으로도 변형 가능(이건 그냥 내 스타일이어서 취사 선택 해주시길 바람!)